### ISSUE TITLE
[Screen Reader-Projects-My Stuff-Import]: Label is not defined for the Browse Search box showing on Import dialog box.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2544]
**Pre-Requisite:** Turn on Narrator(Win + Ctrl + Enter)

**User Experience:** 
Users who depends on Screen Reader will get confuse if Label is not defined for the Browse Search box then it would difficult for the Screen reader users to access the Browse Search box .

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool Used: IE11 Developer (F12)

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Projects control lying in the header section on the page and select it.
4. Navigate to the MY Stuff tab on the page opened and select Import option.
5. Navigate to various controls lying on the page opened.
6. Verify that Label is defined for the Browse Search box or not.
 
**Actual Result:-**
Label is not defined for the Browse Search box showing on Import dialog box.

**Expected Result:-**
Proper Label should be defined for the Browse Search box showing on Import dialog box.

**Note:** This issue exists with NVDA +FF.
 
**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}

**Please refer the attachment.**
![mas3 3 2_projects_import file](https://user-images.githubusercontent.com/30459830/28666110-ab98f54a-72e3-11e7-9627-ec95c8228291.jpg)

**Comment by veler** [comment]
@v-amik Issue fixed and available to test

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug
